hoomd/mpcd/pytest/test_geometry.py

```diff
@@ -19,6 +19,50 @@ def snap():
     return snap_
 
 
+class ConcentricCylinders:
```

Comment:
This test class will not run without `Test` as a prefix, I think.
```suggestion
class TestConcentricCylinders:
```